packages/desktop-client/src/components/schedules/SchedulesTable.js

```diff
@@ -99,7 +104,7 @@ export function ScheduleAmountCell({ amount, op }) {
             lineHeight: '1em',
             marginRight: 10
           }}
-          title={(isApprox ? 'Approximately ' : '') + str}
+          title={(isApprox ? t('schedules.approximately') : '') + str}
```

Comment:
```suggestion
          title={isApprox ? `${t('schedules.approximately') }` : '') + str}
```
We're missing the trailing space here.

Reply:
@TomAFrench 
I think will be better:
```js
title={t('schedules.approximately') + ' ' + str}
```

At this point, will always be approx. Beside, doing ``` `${t('schedules.approximately') }` ``` will fail with lint

Reply:
Could be:
```js
{(isApprox ? `${t('schedules.approximately')} ` : '') + str}
```
to avoid lint

Reply:
Sorry, yes that second one is what I meant to type.